### PR TITLE
Json settings modified for pull request

### PR DESCRIPTION
**Pull request by cmgc**
Google auth was added. + some minor changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,6 @@
 2. ```vagrant up```
 3. ```vagrant ssh```
 4. ```cd /project_data/```
-5. ```CONFIG_CLASS='api.v1_0.settings.LocalSettings' fab init_db``` for first time usage.
-6. ```CONFIG_CLASS='api.v1_0.settings.LocalSettings' ./app.py``` to start app.
+5. ```fab init_db``` for first time usage.
+6. ```./app.py``` to start app.
 7. go to localhost:8001 in your browser
```
Comment (ITsvetkoFF) [inline README.md]: please change 8001 to a correct number :)

File: README.md [hunk 2]
```diff
@@ -4,6 +4,6 @@
 2. ```vagrant up```
 3. ```vagrant ssh```
 4. ```cd /project_data/```
```
Comment (ITsvetkoFF) [inline README.md:6]: the note is that this folder "does not exist" till you cd /project_data/

File: Vagrantfile [hunk 3]
```diff
@@ -9,7 +9,7 @@ Vagrant.require_version ">= 1.7.0"
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
```
Comment (ITsvetkoFF) [inline Vagrantfile:9]: please provide the new vagrant file and check if the private key is in place

File: ecomap/api/v1_0/handlers/auth.py [hunk 4]
```diff
@@ -0,0 +1,175 @@
+import functools
+import tornado.web
+import tornado.auth
+import tornado.escape
+
+from api.v1_0.handlers.base import BaseAPIHandler
+
+try:
+    import urllib.parse as urllib_parse  # py3
```
Comment (ITsvetkoFF) [inline ecomap/api/v1_0/handlers/auth.py]: but we use py2... and not writing the code fully compliant to 3...

File: README.md [hunk 5]
```diff
@@ -4,6 +4,6 @@
...
-5. ```CONFIG_CLASS='api.v1_0.settings.LocalSettings' fab init_db``` for first time usage.
-6. ```CONFIG_CLASS='api.v1_0.settings.LocalSettings' ./app.py``` to start app.
+5. ```fab init_db``` for first time usage.
+6. ```./app.py``` to start app.
```
Comment (ITsvetkoFF) [inline README.md]: for me this syntax does not work. I need to write:

```
python app.py
```

File: ecomap/api/v1_0/handlers/auth.py [hunk 6]
```diff
@@ -0,0 +1,175 @@
...
+        self.write("""\
+        <html>
+        <body>
+        <form action="/auth/login" method="POST">
+        Email: <input type="email" name="email"><br>
+        Password: <input type="password" name="password"><br>
+        <input type="submit" value="Sign in"><br>
+        </form>
+        <p>Sign in with <a href="/auth/login/facebook">Facebook</a> or
+        <a href="#">Gmail</a></p>
```
Comment (ITsvetkoFF) [inline ecomap/api/v1_0/handlers/auth.py]: we can add google link here

File: ecomap/api/v1_0/handlers/auth.py [hunk 7]
```diff
@@ -0,0 +1,175 @@
...
+        self.redirect(self.settings['default_redirect'])
+
+
+
+#  Google auth
+
+class AuthError(Exception):
+    pass
+
+class GoogleLoginHandler(BaseAPIHandler, tornado.auth.GoogleOAuth2Mixin):
```
Comment (ITsvetkoFF) [inline ecomap/api/v1_0/handlers/auth.py:106]: can you please write some comments to your functions in this class? :)

File: ecomap/api/v1_0/urls/__init__.py [hunk 8]
```diff
@@ -0,0 +1 @@
+__author__ = 'yevhen'
```
Comment (ITsvetkoFF) [inline ecomap/api/v1_0/urls/__init__.py]: not so useful info :)
we all know who is who

**Comment by ITsvetkoFF** [comment]
and the last and general comment: please squash everything to one commit :)

File: README.md (hunk 2 above)
Comment (yevhen-m) [inline README.md:6]: Serhiy, why don't we name our source folder on the virtual machine "ecomap" instead of "project_data" to be consistent?

Comment (cmgc) [inline README.md:6]: Yevhen, 'project_data' name does not linked to any project name and was added when we did not have any project template. 
However, you can rename it in whatever you want in Vagrantfile (please write it to Andrey, and he will update file).

Comment (yevhen-m) [inline README.md:6]: Ok, thanks.

Comment (yevhen-m) [inline README.md:6]: Кстати, у тебя уже началась стажировка?

П.С. Я правильно понимаю, что тебе не нужна помощь в доработке ветки по
комментариям Игоря? Если нужна, дай знать, помогу чем смогу.

**Comment by cmgc** [comment]
All fixed except the Vagrantfile. Will be updated by Andrey.

**Comment by ghost** [comment]
Vagrantfile was updated manually :)

**Comment by ITsvetkoFF** [comment]
what about squashing everything into one commit (not 11)? :)

**Comment by cmgc** [comment]
As I can see it were previous commits. The last mine are "Commits on May 8, 2015
@cmgc
fixes"
and Vagrantfile commit on May 9.

p.s. Did you mean create a new branch for pull request, and add all fixes to it?

**Comment by ITsvetkoFF** [comment]
yep :) new branch with all these 11 commits squashed.

**Pull request closed without merging**